src/OSPSuite.SimModelNative/src/CppODEExporter.cpp

```diff
@@ -175,6 +173,18 @@ namespace SimModelNative
 				for (int i = 0; i < sim->Parameters().size(); i++)
 					sim->Parameters()[i]->SetIsFixed(false);
 			}
+         else
+         {
+            //TODO workaround
+            //Observers created for persistable parameters are not properly exported
+            for (int i = 0; i < sim->Parameters().size(); i++)
+            {
+               auto parameter = sim->Parameters()[i];
+               if (parameter->IsPersistable())
+                  parameter->SetIsFixed(false);
+            }
+         }
+
```

Comment:
workaround, no idea why persistable parameters are not properly exported.
C++ export code is not easy to understand - really needs to be refactored one day